plutus-streaming/src/Index/TxIdStatus.hs

```diff
@@ -0,0 +1,92 @@
+{-# LANGUAGE NamedFieldPuns    #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Index.TxIdStatus where
```

Comment:
You put this as part of plutus-streaming, should it be in a separate package? called marconi? @koslambrou what should we do here?